data/scenarios/04Testing/428-drowning-destroy.yaml

```diff
@@ -0,0 +1,60 @@
+name: Drowning results in destruction
+description: |
+  Destroying robots doesn't work.
+  https://github.com/swarm-game/swarm/issues/428
+win: |
+  def isAlive = \n. try {
+    _ <- robotNamed n;
+    log ("Is alive: " ++ n);
```

Comment:
I don't think those logs will be considered (even if they contained "Fatal error") because there is a new robot with id 0 created for this purpose every tick and it is not in the robot map. Can you see them anywhere?

But I guess they are useful when trying this manually.

Reply:
Indeed the winning log is not visible.